docs/en/observability/whats-new.asciidoc

```diff
@@ -3,19 +3,248 @@
+
+The improved APM service overview page provides you with: 
+
+* High-level details including service, runtime and framework versions, container
+and Kubernetes information, along with cloud metadata, if applicable
+* Latency timeline overlapped with deployment events, alert violations, and anomalies
+which occurred during the same period.
+* Top contributors grouped by separate dimensions:
+** Transactions - quickly identify which transactions are impacted and drill down for details.
+** Errors - correlate the rate of errors and exceptions and their start time to the service experience.
+** Backends - understand which services and backends you depend on and how they impact your service performance.
+** Instances - identify whether the problem is occurring on specific instances or across your entire cluster.
+
+For more details, see {kibana-ref}/service-overview.html[Service overview].
+
+[discrete]
+== APM and logs
+
+Inspecting application log data and contextual traces can be extremely helpful to
+troubleshoot a performance problem quickly. Since 7.5, the {apm-app} has offered an
+easy navigation option that allows you to navigate from your distributed traces into the logs.
+
+For {minor-version}, we have two exciting announcements:
+
+* Elastic Common Schema (ECS) logging libraries are now generally available! You can now use
+ECS logger plugins to format your logs into ECS-compatible JSON. Using these libraries, along
+with an Elastic APM agent, means that you do not have any manual configuration to correlate
+logs to your application traces.
+* Embedded trace logs are now available in the curated {apm-app}, which means you no longer have
+to navigate outside the trace view to visualize your application logs. These contextualized
+and correlated logs quickly point you to the underlying issues within your application.
+
+[role="screenshot"]
+image::images/apm-logs.png[Logs within APM]
+
+[discrete]
+== Enhanced host details view
+
+Monitoring large infrastructures can be a complicated and time-consuming task. To get to the
+root cause of a problem, you may need to analyze multiple data streams from various
+infrastructure parts and keep track of related information. To help you minimize the time
+it takes to troubleshoot a problem, you want quick access to all the relevant details on
+the host, including the main KPIs, processes running on the host, logs originating from it,
+and other detailed host information.
+
+In {minor-version}, we’re delighted to introduce an enhanced view of hosts or VMs to the
+Metrics app. Without leaving the Inventory page, you can view the following information about the host or VM:
+
+* Time charts of key host metrics: CPU, memory, load, and network
+* Logs generated by the host or the services running on the host
+* Top processes running on the host by CPU or memory
+* Host metadata: host, cloud, and agent information
+
+The enhanced host details help accelerate root cause analysis by tightly aligning how the
```

Comment:
would be better to break this into two sentences. Otherwise, I think the subj/verb agreement is off.

It's a little hard to parse the two clauses.